lib/plugins/aws/package/compile/events/eventBridge/index.js

```diff
@@ -144,6 +168,32 @@ class AwsCompileEventBridgeEvents {
               );
             }
 
+            if (RetryPolicy) {
+              if (!shouldUseCloudFormation) {
+                throw new ServerlessError(
+                  'Configuring RetryPolicy is not supported for EventBrigde integration backed by Custom Resources. Please use `provider.eventBridge.useCloudFormation` setting to use native CloudFormation support for EventBridge.',
+                  'ERROR_INVALID_RETRY_POLICY_TO_EVENT_BUS_CUSTOM_RESOURCE'
+                );
+              }
+              RetryPolicy = _.mapKeys(
```

Comment:
I think here it would be better to be explicit than to transform the keys this way - it's less readable in my opinion and there are only two properties to assign + we're trying to limit `lodash` usage as much as possible